test/0.8.9/unstructured-storage.test.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "forge-std/Test.sol";
+import { UnstructuredStorage } from "contracts/0.8.9/lib/UnstructuredStorage.sol";
+
+contract ExposedUnstructuredStorage {
+    function getStorageBool(bytes32 position) public view returns (bool) {
+        return UnstructuredStorage.getStorageBool(position);
+    }
+
+    function getStorageAddress(bytes32 position) public view returns (address) {
+        return UnstructuredStorage.getStorageAddress(position);
+    }
+
+    function getStorageBytes32(bytes32 position) public view returns (bytes32) {
+        return UnstructuredStorage.getStorageBytes32(position);
+    }
+
+    function getStorageUint256(bytes32 position) public view returns (uint256) {
+        return UnstructuredStorage.getStorageUint256(position);
+    }
+
+    function setStorageBool(bytes32 position, bool data) public {
+        return UnstructuredStorage.setStorageBool(position, data);
+    }
+
+    function setStorageAddress(bytes32 position, address data) public {
+        return UnstructuredStorage.setStorageAddress(position, data);
+    }
+
+    function setStorageBytes32(bytes32 position, bytes32 data) public {
+        return UnstructuredStorage.setStorageBytes32(position, data);
+    }
+
+    function setStorageUint256(bytes32 position, uint256 data) public {
+        return UnstructuredStorage.setStorageUint256(position, data);
+    }
+}
+
+contract ExposedUnstructuredStorageTest is Test {
```

Comment:
It's excellent by all means :+1: 

I'd suggest only fuzzing the `position` value to make the test unconditionally exhaustive :100:

Reply:
good idea, will do